README.rst

```diff
@@ -91,10 +91,10 @@ and so you will need to add some ``sudo`` commands to the instructions below.
         conda env create -f xc7/environment.yml
         conda activate xc7
         mkdir -p $INSTALL_DIR/xc7/install
-        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/presubmit/install/1049/20201123-030526/symbiflow-arch-defs-install-05bd35c7.tar.xz | tar -xJC $INSTALL_DIR/xc7/install
-        mkdir -p $INSTALL_DIR/xc7/install/share/symbiflow/arch
-        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/presubmit/install/1049/20201123-030526/symbiflow-xc7a50t_test.tar.xz | tar -xJC $INSTALL_DIR/xc7/install/share/symbiflow/arch
-        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/presubmit/install/1049/20201123-030526/symbiflow-xc7a100t_test.tar.xz | tar -xJC $INSTALL_DIR/xc7/install/share/symbiflow/arch
+        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/112/20201208-080919/symbiflow-arch-defs-install-7c1267b7.tar.xz | tar -xJC $INSTALL_DIR/xc7/install
```

Comment:
We should probably move `112/20201208-080919/` into an environment value or something?

Reply:
Can you log a bug about `symbiflow-arch-defs-install-7c1267b7.tar.xz` should probably be using a `git-describe` instead of the hash directly in the name.

Reply:
We could, but it would still be temporary until we can get to use the latest link, which should happen soon. If you prefer though we might enclose the whole `/https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/112/20201208-080919` line into a var and use it for all the install packages.

Reply:
> We should probably move `112/20201208-080919/` into an environment value or something?

Done

Reply:
Did you forget to push or was the resulted quote different?

Reply:
Right, different quote actually, I have opened the issue in archdefs to use git-describe